exp-player/addon/components/exp-page-number/template.hbs

```diff
@@ -0,0 +1 @@
+<p style="width:100%; text-align: right">{{pageNumber}}</p>
```

Comment:
In general we try to avoid use of hardcoded styles in templates, but I realize this is kind of an unusual situation...

Note solely for future reference... ember components are rendered in an outer div which is slightly unnecessary here. This can be controlled in component JS, eg with the `tagName` and `classNames` properties. (normally I wouldn't bother with those, but for a simple component like this it might be useful)